src/main/app/src/Components/HomeComponent/Home.tsx

```diff
@@ -0,0 +1,230 @@
+import React, {useState, useEffect, useContext, useCallback, ReactElement} from 'react';
+
+import {ILandscape} from '../../interfaces';
+import GenericModal from '../ModalComponent/GenericModal';
+import LandscapeLog from '../LandscapeComponent/Log/LandscapeLog';
+import Command from '../CommandComponent/Command';
+import {Link} from 'react-router-dom';
+import {createStyles, Theme, makeStyles} from '@material-ui/core/styles';
+import CommandContext from '../../Context/Command.context';
+import LandscapeContext from '../../Context/Landscape.context';
+import Typography from '@material-ui/core/Typography';
+import MenuIcon from '@material-ui/icons/Menu';
+import Grid from '@material-ui/core/Grid';
+import {Button, Fab} from '@material-ui/core';
+import './Home.scss';
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        marginBottom: {
+            marginBottom: theme.spacing(1),
+        },
+        extendedIcon: {
+            marginRight: theme.spacing(1),
+        },
+    }),
+);
+
+const Home: React.FC = () => {
+    const [modalContent, setModalContent] = useState<string | ReactElement | ReactElement[] | null>(
+        null
+    );
+    const [landscapes, setLandscapes] = useState<ILandscape[]>();
+    const [loadLandscapes, setLoadLandscapes] = useState<boolean>(true);
+
+    const commandContext = useContext(CommandContext);
+    const landscapeContext = useContext(LandscapeContext);
+    const classes = useStyles();
+    //Could be moved into useEffect but can be used for a reload button later on
+    const getLandscapes = useCallback(async () => {
```

Comment:
This could move to the LandscapeContext and in general REST calls could go to the corresponding contexts. By doing so, the components have less responsibilities and can concentrate on doing what they know how to do. In this case Home is really big already (see below), but still, even if it was smaller, it should only know how to do "Home" :)